src/node/hooks.h

```diff
@@ -119,4 +120,49 @@ namespace ccf
     }
   };
 
+  // Note: The SignaturesHook and SerialisedMerkleTreeHook are separate because
+  // the signature and the Merkle tree are recorded in distinct tables (for
+  // serialisation performance reason). However here, they are expected to
+  // always be called one after the other and for the same version.
```

Comment:
Perhaps clarify that this is to do with the sorting order of the table names?

Reply:
In this case, this isn't about the naming of the table but the fact that both tables are always written to at the same time.